experimental/packages/opentelemetry-configuration/src/FileConfigProvider.ts

```diff
@@ -254,7 +254,16 @@ function getConfigHeaders(
   return null;
 }
 
-function parseConfigExporter(exporter: SpanExporter): SpanExporter {
+enum ProviderType {
+  TRACER = 0,
+  METER = 1,
+  LOGGER = 2,
+}
+
+function parseConfigExporter(
+  exporter: SpanExporter,
+  providerType: ProviderType
+): SpanExporter {
```

Comment:
Since we are now using this for multiple signals, do we need to return different exporters for each signal (e.g. LogExporter vs SpanExporter)?